libm/riscv64/fenv.c

```diff
@@ -95,6 +97,12 @@ int fegetround(void)
 
 int fesetround(int round)
 {
+  /* Check whether requested rounding direction is supported
+   * Currently, we deem FE_TONEAREST_MM (RMM) as invalid
+   * should any usecase for that arise, we can simply change the mask */
+  if (round & ~RVF_ROUND_MASK) {
```

Comment:
if we add RMM later, RVF_ROUND_MASK would be 111b and this makes `round & ~RVF_ROUND_MASK` failed to detect 101b/110b/111b.

suggest directly judge with  `if ( round >= FE_TONEAREST && round <= FE_UPWARD)` for valid cases.